src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react';
+import cn from 'classnames';
+import { ErrorMessage } from '../types/ErrorMessage';
+
+type Props = {
+  error: string;
+  setErrorMessage: (error: ErrorMessage) => void;
+};
+
+export const ErrorNotification: React.FC<Props> = props => {
+  const { error, setErrorMessage } = props;
+
+  useEffect(() => {
+    if (error === ErrorMessage.Default) {
+      return;
+    }
+
+    const timer = setTimeout(() => {
+      setErrorMessage(ErrorMessage.Default);
+    }, 3000);
+
+    return () => clearTimeout(timer);
+  }, [error]);
```

Comment:
React Hook useEffect has a missing dependency: 'setErrorMessage'. Either include it or remove the dependency array